resources/ottawa.csv

```diff
@@ -1,7 +1,7 @@
-capabilities,"vm:standard,arch:32bit,os:win,arch:x86,test:java9","vm:standard,arch:64bit,os:win,arch:x86,test:java9","vm:compressed_refs,arch:64bit,os:win,arch:x86,test:java9",,"vm:standard,arch:32bit,os:linux,os:unix,arch:x86,test:java9","vm:standard,arch:64bit,os:linux,os:unix,arch:x86,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:x86,test:java9","vm:mixed_refs,arch:64bit,os:linux,os:unix,arch:x86,test:java9","vm:standard,arch:32bit,os:linux,os:unix,arch:ppc,test:java9","vm:standard,arch:64bit,os:linux,os:unix,arch:ppc,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:ppc,test:java9","vm:standard,arch:64bit,os:linux,os:unix,arch:ppcle,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:ppc,arch:ppcle,test:java9",,"vm:standard,arch:31bit,os:linux,os:unix,arch:390,test:java9","vm:standard,arch:64bit,os:linux,os:unix,arch:390,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:390,test:java9",,"vm:standard,arch:32bit,os:aix,os:unix,arch:ppc,test:java9","vm:standard,arch:64bit,os:aix,os:unix,arch:ppc,test:java9","vm:compressed_refs,arch:64bit,os:aix,os:unix,arch:ppc,test:java9",,"vm:standard,arch:31bit,os:zos,arch:390,test:java9","vm:standard,arch:64bit,os:zos,arch:390,test:java9","vm:compressed_refs,arch:64bit,os:zos,arch:390,test:java9","arch:32bit,os:linux,arch:arm","vm:standard,arch:64bit,os:osx,os:unix,arch:x86,test:java9","vm:compressed_refs,arch:64bit,os:osx,os:unix,arch:x86,test:java9",,"vm:compressed_refs,arch:64bit,os:bsd,os:unix,arch:x86,test:java9","vm:compressed_refs,arch:64bit,os:sunos,os:unix,arch:x86,test:java9","vm:standard,arch:64bit,os:linux,os:unix,arch:aarch64,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:aarch64,test:java9",,"vm:standard,arch:64bit,os:linux,os:unix,arch:riscv,test:java9","vm:compressed_refs,arch:64bit,os:linux,os:unix,arch:riscv,test:java9"
+,win_x86,win_x86-64_nocmprssptrs,win_x86-64_cmprssptrs,win_x86-64,linux_x86,linux_x86-64_nocmprssptrs,linux_x86-64_cmprssptrs,linux_x86-64,linux_ppc,linux_ppc-64_nocmprssptrs,linux_ppc-64_cmprssptrs,linux_ppc-64_nocmprssptrs_le,linux_ppc-64_cmprssptrs_le,linux_ppc-64_le,linux_390,linux_390-64_nocmprssptrs,linux_390-64_cmprssptrs,linux_390-64,aix_ppc,aix_ppc-64_nocmprssptrs,aix_ppc-64_cmprssptrs,aix_ppc-64,zos_390,zos_390-64_nocmprssptrs,zos_390-64_cmprssptrs,linux_arm,osx_x86-64_nocmprssptrs,osx_x86-64_cmprssptrs,osx_x86-64,bsd_x86-64,sunos_sparcv9-64,linux_aarch64_nocmprssptrs,linux_aarch64_cmprssptrs,linux_aarch64,linux_riscv64_nocmprssptrs,linux_riscv64_cmprssptrs

```

Comment:
@smlambert I assume that bsd_x86-64 and sunos_sparcv9-64 are all mixed, correct?

Reply:
yes, assume those are mixed, thanks 👍